lm_eval/few_shot_examples/codexglue_text_to_text_few_shot_prompts.json

```diff
@@ -0,0 +1 @@
+{"danish":{"source1":"2 . Udfyld felterne i hvert trin i vejledningen . ","source2":"* Vise rapporter med finansposter og saldi . ","target1":"2 . Fill in the fields in each step of the guide . ","target2":"* View reports that show general ledger entries and balances . "},"chinese":{"source1":"返回 与 筛选器 初始化 由 平台 的 MCDRemoteSystemPlatformFilter 对象 。 ","source2":"用于 将 本地 的 （ 调用 ） 应用 程序 可 见性 首选 项 设置 发现 远程 系统 时 的 类 。 ","target1":"Returns an MCDRemoteSystemPlatformFilter object initialized with a filter by platform . ","target2":"A class used to set the local ( calling ) application visibility preference when discovering remote systems ."},"norwegian":{"source1":"Kosttypesaldo = Kostsentersaldo + Kostobjektsaldo ","source2":"* Vise en liste over bokføringsgrupper som du posterer til kontoen . ","target1":"Cost Type Balance = Cost Center Balance + Cost Object Balance ","target2":"* See a list of posting groups that post to that account . "},"latvian":{"source1":"# # &lt; a name = &quot; 6-change-the-status-of-the-conversion-record-to-ready &quot; &gt; &lt; / a &gt; 6 . Mainiet pārveidošanas ieraksta statusu uz Gatavs ","source2":"# # &lt; a name = &quot; update-the-list-of-accounts-that-are-included-in-the-list-of-transactions &quot; &gt; &lt; / a &gt; Transakciju sarakstā ietvertā kontu saraksta atjaunināšana ","target1":"# # 6 . Change the status of the conversion record to Ready ","target2":"# # Update the list of accounts that are included in the list of transactions "}}
```

Comment:
we don't need this file now that we have it inside `tasks/few_shot_examples`

Reply:
Thanks, I have removed the file.